### PR TITLE
[zh-cn] sync labels-annotations-taints/_index.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1110,25 +1136,38 @@ If the number of backend endpoints falls below 1000, the control plane removes t
...
 Used on: Jobs
 
 The presence of this annotation on a Job indicates that the control plane is
 [tracking the Job status using finalizers](/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers).
+The control plane uses this annotation to safely transition to tracking Jobs
+using finalizers, while the feature is in development.
 You should **not** manually add or remove this annotation.
 -->
-### batch.kubernetes.io/job-tracking {#batch-kubernetes-io-job-tracking}
+### batch.kubernetes.io/job-tracking (deprecated) {#batch-kubernetes-io-job-tracking}
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:1151]: If `(deprecated)` is added, the localization is required.

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -1110,25 +1136,38 @@ If the number of backend endpoints falls below 1000, the control plane removes t
 如果后端端点的数量低于 1000，则控制平面将移除此注解。
 
 <!--
-### batch.kubernetes.io/job-tracking
+### batch.kubernetes.io/job-tracking (deprecated) {#batch-kubernetes-io-job-tracking}
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:1139]: If `(deprecated)` is added, the localization is required.

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -901,6 +901,32 @@ ServiceAccount that the token (stored in the Secret of type `kubernetes.io/servi
...
+If a legacy token was last used before the cluster gained the feature (added in Kubernetes v1.26), then
+the label isn't set.
+-->
+### kubernetes.io/legacy-token-last-used
+
+例子：`kubernetes.io/legacy-token-last-used: 2022-10-24`
+
+用于：Secret
+
+控制面仅为带有 `kubernetes.io/service-account-token` 类的 Secret 添加此标签。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:924]:
```suggestion
控制面仅为 `kubernetes.io/service-account-token` 类型的 Secret 添加此标签。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -901,6 +901,32 @@ ServiceAccount that the token (stored in the Secret of type `kubernetes.io/servi
...
+该标签的值会记录控制面最后一次看到客户端使用服务帐户令牌进行身份验证的请求的日期（ISO 8601
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:925]: Line 925 读上去很别扭，需要再想想，另外`时域` -> `时区`

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 5]
```diff
@@ -901,6 +901,32 @@ ServiceAccount that the token (stored in the Secret of type `kubernetes.io/servi
...
+格式，UTC 时域）
+
+如果在集群获得该特性（添加于 Kubernetes v1.26）之前最后一次使用了旧版令牌，则不会设置标签。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:928]:
```suggestion
如果上一次使用老的令牌的时间在集群获得此特性（添加于 Kubernetes v1.26）之前，则不会设置此标签。
```

**Pull request merged**